src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionValidatorTest.java

```diff
@@ -16,429 +16,462 @@
 */
```

Comment:
The indention in this file has become messed up, no other edits, please make sure your editor don´t changes files you do not work in, this file should if possible be reverted to it´s original state.